metadata/disclosures/prebid/tld-probe.json

```diff
@@ -0,0 +1,20 @@
+{
+  "disclosures": [
```

Comment:
what about https://github.com/prebid/Prebid.js/blob/4d01a5e2576dafb167ceb45bff37498d4a6a49dc/src/storageManager.js#L130

Reply:
good catch. both this and debugging do not go through storageManager so I missed them (and they wouldn't generate warnings), but it's still good to disclose them.